event_attend.html

```diff
@@ -105,7 +111,7 @@ <h1 id="page-title" class="title3 text-style-title3 text-center">{{ event.title
 		{% if campaign.show_address1 and page.custom_fields.events_webinar != 'Yes'   %}
 				<span class="event-address1">{{ event.address1 }}</span><br>
 		{% endif %}
-		{{page.custom_fields.events_webinar != 'Yes' }}

```

Comment:
Just confirming this was meant to be removed, right?

Reply:
yes, it was breaking the page